src/components/interfaces/HMI_API.xml

```diff
@@ -1579,8 +1584,11 @@
 <!-- Remote Control -->
 <enum name="ModuleType">
    <element name="CLIMATE"/>
-   <element name="RADIO"/>
    <element name="SEAT"/>
+   <element name="RADIO"/>
```

Comment:
Should not rearrange existing order of `ModuleType`s

Reply:
@jacobkeeler rearrange removed